### PR TITLE
Update greetings and origin comments

### PR DESCRIPTION
**Pull request by drewvolz**
* Adds 10 new greetings
* Updates origin comments

### CODE REVIEW

File: modules/gob-web/modules/student/student-summary.js [hunk 1]
```diff
@@ -44,6 +45,15 @@ const welcomeMessages = [
...
 	'Salve, ', // latin
 	'Χαῖρε! ', // ancient greek
+	'Zdravo, ', // bosnian
+	'Bok, ', // croatian
+	'ahoj, ', // czech, slovak
+	'Tere, ', // estonian
+	'Bula, ', // fijian
+	'Zdravo, ', // serbian
+	'Hujambo, ', // swahili
+	'Xin chào, ', // vietnamese
```
Comment (hawkrives) [inline modules/gob-web/modules/student/student-summary.js]: Is this the right comma for Vietnamese? I know Korean/Japanese/Chinese all use non-english commas, if they use commas at all.

Comment (drewvolz) [reply modules/gob-web/modules/student/student-summary.js]: Everywhere I look points to this comma.

**Comment by hawkrives** [comment]
see, I can see that you rebased/force pushed/something, but now I can't see what got changed, if anything. did you change anything?

File: modules/gob-web/modules/student/student-summary.js [hunk 2]
```diff
@@ -27,13 +27,14 @@ const welcomeMessages = [
 	'Hello, ',
 	'こんにちは、', // japanese
 	'ようこそ、', // japanese
-	'Fram! Fram! ',
-	'Salut, ',
+	'Fram! Fram! ', // new norwegian
```
Comment (hawkrives) [inline modules/gob-web/modules/student/student-summary.js]: should we add proper norwegian?

should we change this to "olaf"?

Comment (drewvolz) [reply modules/gob-web/modules/student/student-summary.js]: [source](https://en.wikipedia.org/wiki/St._Olaf_College#Seal_and_motto)
> The motto Fram! Fram! Kristmenn, Krossmenn, written in New Norwegian

Comment (drewvolz) [reply modules/gob-web/modules/student/student-summary.js]: I'd also accept [Nynorsk](https://en.wikipedia.org/wiki/Nynorsk)

**Pull request merged**